webclient/index.html

```diff
@@ -125,6 +139,8 @@
                 alert('Whoops. The cached files on your machine are out of sync with each other. That\'s our bad. Please hard-refresh the page?');
             }
         </script>
+<flip-card id="mainFlipCard">
```

Comment:
Hm. That sounds like an ugly layout problem and I'm not sure it'd work out very well well in practice. I'm guessing you think it's important to keep the inputs simultaneously visible with the summary table? Or would a "back" button suffice?